File: Goal.cpp
#include<iostream>
#include<string>
#include<cstdio>
using namespace std;

class Goal{
private:
	string time;
	int match_id;
	string goalerTeamCode;
	int goaler_id;
	bool ifOG;
	string goal_belong;
public:
	Goal(string time, int match_id, string goalerTeamCode, int goaler_id, bool ifOG, string goal_belong);
	string get_time();
	int get_match_id();
	string get_goalerTeamCode();
	int get_goaler_id();
	bool get_ifOG();
	string get_goal_belong();
};

Goal::Goal(string time, int match_id, string goalerTeamCode, int goaler_id, bool ifOG, string goal_belong){
	this -> time = time;
	this -> match_id = match_id;
	this -> goalerTeamCode = goalerTeamCode;
	this -> goaler_id = goaler_id;
	this -> ifOG = ifOG;
	this -> goal_belong = goal_belong;
};

string Goal::get_time(){
	return this -> time;
}

int Goal::get_match_id(){
	return this -> match_id;
}

string Goal::get_goalerTeamCode(){
	return this -> goalerTeamCode;
}

int Goal::get_goaler_id(){
	return this -> goaler_id;
}

bool Goal::get_ifOG(){
	return this -> ifOG;
}

string Goal::get_goal_belong(){
	return this -> goal_belong;
}

int main(){
	cout << "hello";
}

File: Python/Goal.py
import json
from Overall_func import *
# ["2018-06-14 16:26:00", 0, "HOME", 22, false, "HOME"]


class Goal:
	"""docstring for Goal"""

	# The string input herer has to be attached with '[]'
	def __init__(self, string):
		self.info = json.loads(string)
		self.time = date_intp(self.info[0])
		self.match_id = self.info[1]
		self.goaler_team = self.info[2]
		self.goaler_id = self.info[3]
		self.ifOG = self.info[4]
		self.goal_belong = self.info[5]

	def get_time(self):
		return self.time

g = Goal('["2018-06-14 16:26:00", 0, "HOME", 22, false, "HOME"]')
print(g.get_time())